frontend/components/product-list/sections/FilterableProductsSection/Pagination.tsx

```diff
@@ -0,0 +1,142 @@
+import { HStack, useBreakpointValue } from '@chakra-ui/react';
+import {
+   Pagination as BasePagination,
+   PaginationItem,
+   PaginationLink,
+   useIsMounted,
+} from '@ifixit/ui';
+import * as React from 'react';
+import {
+   HiChevronDoubleLeft,
+   HiChevronDoubleRight,
+   HiChevronLeft,
+   HiChevronRight,
+} from 'react-icons/hi';
+import {
+   usePagination,
+   UsePaginationProps,
+} from 'react-instantsearch-hooks-web';
+
+type PaginationProps = UsePaginationProps;
+
+export function Pagination(props: PaginationProps) {
+   const { currentRefinement, refine, createURL, nbPages } =
+      usePagination(props);
+   const responsiveVisibleNumberOfPages = useBreakpointValue({
+      base: 3,
+      sm: 5,
+      lg: 7,
+   });
+   const isMounted = useIsMounted();
+   const visibleNumberOfPages = isMounted ? responsiveVisibleNumberOfPages : 3;
+
+   const createPageUrl = (page: number) => {
+      try {
+         const fullUrl = createURL(page - 1);
+         const url = new URL(fullUrl);
+         return url.pathname + url.search;
+      } catch (error) {
```

Comment:
Hmm, what kind of errors were we expecting here?

Reply:
It was throwing due to Algolia routing misconfiguration. I have fixed it, so I think it's safe now to drop the try catch 👍